bot/commands/recommend.py

```diff
@@ -1,13 +1,15 @@
 """Standard commands for recommending content to other users."""
 from discord import Embed, Member
 from discord.ext import commands
+from typing import Union
```

Comment:
Same comment re: grouping of imports (`typing` is from the stdlib so it must be on it's own group above the second-party import group)
```suggestion
from typing import Union

from discord import Embed, Member
from discord.ext import commands

```